applications/tari_validator_node/src/consensus/spec.rs

```diff
@@ -26,6 +31,10 @@ pub struct TariConsensusSpec;
 
 impl ConsensusSpec for TariConsensusSpec {
     type Addr = PeerAddress;
+    type BlockTransactionExecutorBuilder = TariDanBlockTransactionExecutorBuilder<
```

Comment:
I suspect we won't need a build interface, seems to be just avoiding the borrow checker (immutable self, mutable executor). We can probably just pass in an executor, but not important at this stage.